Platform Documentation.md

```diff
@@ -827,6 +831,9 @@ reminder: python session_reminder.py
 *Note: The WORKER_NAME can be chosen freely, but we recommend to name it in
 a way, that it describes the task of the worker in a short way*
 
+**Don't forget to restart your worker manually after updating your code (push +
+deploy) !**
```

Comment:
s/ !/!/g

Maybe just append that above instead, e.g. "... successful push). Workers have to be redeployed separately ..."